modules/form-field/react/lib/Hint.tsx

```diff
@@ -16,24 +18,27 @@ const Message = styled('p')(type.body2, {width: '100%', margin: `${spacing.xxs}
 
 export default class Hint extends React.Component<HintProps> {
   static ErrorType = ErrorType;
-
+  static defaultProps = {
+    errorLabel: 'Error',
+    alertLabel: 'Alert',
+  };
   public render() {
-    const {children, error} = this.props;
+    const {children, error, errorLabel, alertLabel} = this.props;
 
-    let errorLabel: string | undefined;
+    let hintLabel: string | undefined;
     switch (error) {
       case Hint.ErrorType.Error:
-        errorLabel = 'Error';
+        hintLabel = errorLabel;
         break;
       case Hint.ErrorType.Alert:
-        errorLabel = 'Alert';
+        hintLabel = alertLabel;
         break;
       default:
     }
 
     return (
       <Message {...this.props}>
-        {typeof error !== 'undefined' && <Label>{errorLabel && `${errorLabel}: `}</Label>}
+        {typeof error !== 'undefined' && <Label>{hintLabel && `${hintLabel}: `}</Label>}
```

Comment:
```suggestion
        {typeof error !== 'undefined' && hintLabel && <Label>{hintLabel}: </Label>}
```
A little cleaner and no possibility of an empty `<Label>`